### DISCUSSION TITLE
关于专有名词的一些疑惑

### DISCUSSION THREAD

**Discussion by y761350477** [sparanoid/chinese-copywriting-guidelines#144] [Q&A] [answered]
1. 这是你的 U盘 吗？
2. 我喜欢使用 QQ音乐 听歌。
3. 你使用过 豆瓣FM 吗？
请问以上的 U盘、QQ音乐、豆瓣FM 间空格使用是否符合标准？

**Comment by yoonhakcher** [comment]
你何不使用专名号来标识专名。

**Comment by HumJ0218** [comment]
个人觉得没必要，中文本身没有为专有名词单独添加间隔的传统。另外注意字母和汉字之间的间隔，所以应该写作
1. 这是你的 U 盘吗？
2. 我喜欢使用 QQ 音乐听歌。
3. 你使用过豆瓣 FM 吗？

**Comment by liyishuai** [comment]
英文空格为语义所需，中文与西文加间隔仅为排版所需，语义层面无空格一说。

**Comment by y761350477** [comment]
> 个人觉得没必要，中文本身没有为专有名词单独添加间隔的传统。另外注意字母和汉字之间的间隔，所以应该写作
> 
> 1. 这是你的 U 盘吗？
> 2. 我喜欢使用 QQ 音乐听歌。
> 3. 你使用过豆瓣 FM 吗？

大佬，你好，感谢百忙之中回复，你这样的写法，视觉上不是符合我的审美，
我感觉还是整体表示，阅读起来更舒适
如果更直观如：「U盘」、「QQ音乐」、「豆瓣FM」，但是无疑增加了书写成本，不如空格简便

**Comment by liyishuai** [comment, marked as answer]
想要“整体性”就无视此项目关于空格的指导意见，直接参照新闻出版行业标准CY/T 154—2017号《中文出版物夹用英文的编辑规范》第8.1节：
> 中文文本中夹用英文时，应根据所选用的中英文字体、字符间距以及排版的视觉效果决定英文词句与中文文字之间是否留有空格间距。如留空格，应保证体例的统一。

一律不加空格即可。
